### DISCUSSION TITLE
New Teampass 3.1.1, Master install

### DISCUSSION THREAD

**Discussion by Arash9703** [nilsteampassnet/TeamPass#4059] [Show and tell]
I installed the latest version of TeamPass, 3.1.1. And the lastest commit. I wanted to point out two issues regarding assigning roles to a group. Once a user is created with a role, it becomes impossible to change, remove, or add additional roles to the user. Additionally, during the installation of TeamPass on a LAMP stack server, I encountered errors. Thanks to and @oernii for suggesting that commenting out two scripts in the settings.php file resolved the error. If anyone else encounters errors or has corrections to suggest, it would be appreciated. Thanks! @nilsteampassnet

**Comment by Arash9703** [comment]
[https://github.com/nilsteampassnet/TeamPass/discussions/403]()

**Comment by Arash9703** [comment]
I encountered another error while installing the latest version 3.1.1. I received an AJAX error despite attempting to reinstall multiple times from a clean start. However, the issue persisted without any changes.
![image](https://github.com/nilsteampassnet/TeamPass/assets/143269985/0979f646-896e-4dba-bd81-0fda03d9f73e)
anyone else installing on LAMP stack server got this ! 
Thanks for help

**Reply by morozzkin** [reply]
Same here
modified users.datatable.php from this
`$sOutput .= '"sEcho": '.(int) $query->filter('draw', FILTER_SANITIZE_NUMBER_INT)('draw').', ';  `
to
`$sOutput .= '"sEcho": '.$request->query->filter('draw', FILTER_SANITIZE_NUMBER_INT).', ';`

Users list is loading now, but LDAP sync leads to error 500 anyway

**Reply by g0lgs** [reply]
That change makes no difference for me I still get an Ajax Error.

**Comment by Arash9703** [comment]
![image](https://github.com/nilsteampassnet/TeamPass/assets/143269985/47922342-3c7c-41b7-bae0-b37e0e11e3a4)

**Comment by Arash9703** [comment]
Hello , 

I'm not sure how, but even after altering the default language interface of TeamPass and the default language of the profile menu, it remains in English. The only thing that seems to change is the side menu bar.
Anyone out there who got the same return ! 
![image](https://github.com/nilsteampassnet/TeamPass/assets/143269985/03482349-5212-4355-bdd0-533cc1a57184)

**Reply by Arash9703** [reply]
Now the look has totally changed. 
------------
![image](https://github.com/nilsteampassnet/TeamPass/assets/143269985/632c1a7f-af9c-4690-8af8-af7dc42df6db)
